### PR TITLE
BIGTOP-4296: Add API for cluster and host metrics

### PR DESCRIPTION
**Pull request by Cat-Drink**

### CODE REVIEW

File: bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/utils/ProxyUtils.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            return listNode.get(index).asLong();
+        return 0L;
+    }
+
+    public static JsonNode array2node(double[][] array, int num, int cores) {
+        ObjectMapper mapper = new ObjectMapper();
+        double[] cache = new double[6];
+        for (int i = 0; i < num; i++) for (int j = 0; j < 6; j++) cache[j] += array[i][j];
+        ArrayNode node = mapper.createArrayNode();
+        // 数据排序为日期小在前，日期大在后
```
Comment (wuchunfu) [inline bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/utils/ProxyUtils.java:53]: @Cat-Drink Please modify the comment to English, thanks.

File: bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/proxy/BigtopProxy.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import reactor.core.publisher.Mono;
+
+@Component
+public class BigtopProxy {
+
+    private final WebClient webClient;
+
+    public BigtopProxy(WebClient.Builder webClientBuilder, @Value("http://localhost:5173") String bigTopHost) {
```
Comment (wuchunfu) [inline bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/proxy/BigtopProxy.java:40]: @Cat-Drink It's better not to hard code here, thanks

Comment (Cat-Drink) [reply bigtop-manager-server/src/main/java/org/apache/bigtop/manager/server/proxy/BigtopProxy.java]: ok,i had finished

**Review by wuchunfu — APPROVED** [review]
LGTM

**Pull request merged**